src/main/java/com/google/sps/data/Note.java

```diff
@@ -69,70 +103,70 @@ public String getMessage(){
         return categorizedText;
     }
 
-    /**
-   * Detects words from a picture and returns them
-   * TODO: Clean up the result, sometimes the api misreads, misses entirely, or adds additional words
-   *        
-   */
-  private String detectDocumentText(String path) throws IOException {
-    List<AnnotateImageRequest> requests = new ArrayList<>();
-    //This only works when you publish, api can not read local urls
-    ImageSource imgSource = ImageSource.newBuilder().setImageUri(path).build(); 
-    Image img = Image.newBuilder().setSource(imgSource).build();
-    Feature feat = Feature.newBuilder().setType(Feature.Type.DOCUMENT_TEXT_DETECTION).build();
-    AnnotateImageRequest request = AnnotateImageRequest.newBuilder().addFeatures(feat).setImage(img).build();
-    requests.add(request);
+   /**
+    * Detects words from a picture and returns them
+    * TODO: Clean up the result, sometimes the api misreads, misses entirely, or adds additional words
+    *        
+    */
+    private String detectDocumentText(String path) throws IOException {
```

Comment:
Sorry, this was just formatting -- did not make any changes to `detectDocumentText()`

Reply:
Yeah my formatting was kinda wonky I didn't know if we were going to do double spaces or tabs